tests/end2end/features/downloads.feature

```diff
@@ -30,17 +30,16 @@ Feature: Downloading things from a website.
         And I open data/downloads/issue1243.html
         And I run :hint links download
         And I run :follow-hint a
-        And I wait for "Asking question <qutebrowser.utils.usertypes.Question default='qutebrowser-download' mode=<PromptMode.text: 2> text='Save file to:'>, *" in the log
-        And I run :leave-mode
-        Then no crash should happen
+        And I wait for "Asking question <qutebrowser.utils.usertypes.Question default='qutebrowser-download' mode=<PromptMode.download: 5> text='Save file to:'>, *" in the log
+        Then the error "Download error: No handler found for qute://!" should be shown
```

Comment:
I think the commit message for 99e57a6c07b6fee0cfdd54d49337742575f5c6c7 explains the situation (the first part), basically, the download now starts even without answering the question (because the filename-ask-part was not only in ` .fetch` , but also repeated in `.get`), so the download now fails instead of being cancelled.

I can restore the old(/waiting) behavior, but it seemed kind of unnecessary to repeat the check in `.get` when it's also in `.fetch`.